conductor_api/src/conductor/base.rs

```diff
@@ -504,8 +505,10 @@ impl Conductor {
     pub fn instantiate_from_config(
         &mut self,
         id: &String,
-        config: &Configuration,
+        maybe_config: Option<&Configuration>,
     ) -> Result<Holochain, String> {
+        let self_config = self.config.clone();
+        let config = maybe_config.unwrap_or(&self_config);
```

Comment:
I wonder if this is possible:

```suggestion
        let config = maybe_config.unwrap_or_else(|| &self.config.clone());
```

Reply:
Guess it would be possible, but why clone when we want a reference anyway?